presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -49,6 +51,23 @@ public BaseJdbcConfig setConnectionUrl(String connectionUrl)
         return this;
     }
 
+    @AssertTrue
+    public boolean isConnectionUrlValid()
+    {
+        String connectionUrl = getConnectionUrl();
+        if (!connectionUrl.startsWith("jdbc:")) {
+            return false;
+        }
+
+        try {
+            URI.create(connectionUrl.substring(5));
+            return true;
```

Comment:
I do not know whether this is safe to assume. Looks reasonable, but let's not add this without more testing.

Reply:
Please look at https://github.com/prestosql/presto/pull/6302/files#r541926149